estate/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    'name': 'Estate',
```

Comment:
all of these should be using double quotes
```suggestion
    "name": "Estate",
```